docs/README.md

```diff
@@ -3,7 +3,6 @@ title: Serverless Framework Documentation
 menuText: Docs
 layout: Doc
 menuItems:
-  - {menuText: Getting Started, path: /framework/docs/getting-started/}
```

Comment:
Yes, it is a duplicate.  The docs ALSO create links for each file in the dir.  Since there is a `getting-started.md` file as well as this metadata link, this resulted in duplicate links on our docs site.